eng/pipelines/templates/jobs/ci.yml

```diff
@@ -218,7 +218,7 @@ jobs:
         displayName: 'Run SpotBugs, Checkstyle, RevApi, and Javadoc'
         inputs:
           mavenPomFile: pom.xml
-          options: '$(DefaultOptions) --no-transfer-progress -DskipTests -Dgpg.skip -Dverify-readme -pl $(ProjectList)'
+          options: '$(DefaultOptions) -T 2C --no-transfer-progress -DskipTests -Dgpg.skip -Dverify-readme -pl $(ProjectList)'
```

Comment:
Should hold off on changing this as it will lead to interlaced logging in the console, basically making it impossible to review the logs to determine which linting failed.

There are plans on making this parallelized once logging out linting exceptions to file per artifact is better supported.

Reply:
sounds good. I'll remove it .